src/device-registry/utils/create-airqloud.js

```diff
@@ -502,18 +501,21 @@ const createAirqloud = {
 
         if (responseFromListSites.success === true) {
           const sites = responseFromListSites.data;
-          logObject("sites", sites);
           for (const site of sites) {
-            const { latitude, longitude } = site;
-
-            const isSiteInAirQloud = geolib.isPointInPolygon(
-              { latitude, longitude },
-              airqloudPolygon
-            );
-
-            if (isSiteInAirQloud === true) {
-              site_ids.push(site._id);
-            } else if (isSiteInAirQloud === false) {
+            const { latitude, longitude, _id, description } = site;
+            if (!isEmpty(latitude) && !isEmpty(longitude)) {
```

Comment:
Main part of the PR. In cases where a Site does not have GPS coordinates, just log it. Otherwise, only consider Sites with coordinates for all operations that follow.

**Side related note:**
I have already raised a JIRA support ticket for us to investigate as to why a Site could be without its GPS coordinates yet we have DB validations that check for this. In addition to the endpoint validations! So this is strange but we shall figure it out with time.